src/Stratis.BitcoinD/Properties/launchSettings.json

```diff
@@ -6,6 +6,10 @@
     "Stratis.BitcoinD Test": {
       "commandName": "Project",
       "commandLineArgs": "-testnet"
+    },
+    "Stratis.BitcoinD Test with RPC": {
+      "commandName": "Project",
+      "commandLineArgs": "-testnet -server -rpcpassword=rpcpassword -rpcuser=rpcuser"
```

Comment:
Is this correct to keep in the PR?

Reply:
yeah it's fine, we didn't have one for bitcoin and I needed it. It can stay there.